target/linux/layerscape/image/armv7.mk

```diff
@@ -40,50 +40,41 @@ endef
 TARGET_DEVICES += fsl_ls1021a-twr
 
 define Device/fsl_ls1021a-twr-sdboot
+  $(Device/rework-sdcard-images)
   DEVICE_VENDOR := NXP
   DEVICE_MODEL := TWR-LS1021A
   DEVICE_VARIANT := SD Card Boot
   DEVICE_DTS := ls1021a-twr
-  FILESYSTEMS := ext4
-  IMAGES := sdcard.img sysupgrade.bin
-  IMAGE/sdcard.img := \
-    ls-clean | \
-    ls-append-sdhead $(1) | pad-to 4K | \
-    ls-append $(1)-uboot.bin | pad-to 3M | \
-    ls-append $(1)-uboot-env.bin | pad-to 15M | \
-    ls-append-dtb $$(DEVICE_DTS) | pad-to 16M | \
```

Comment:
By removing this command, we silently drop 
`$(call Image/BuildDTB,$(DTS_DIR)/$(1).dts,$(DTS_DIR)/$(1).dtb)`
from Build/ls-append-dtb

Therefore, build fails with missing dtb for these devices:

FATAL ERROR: Couldn't open "/data/openwrt/build_dir/target-aarch64_generic_musl/linux-layerscape_armv8_64b/linux-5.4.71/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy-sdboot.dtb": No such file or directory

Reply:
Now I used `Image/Prepare` like [archs38](https://github.com/openwrt/openwrt/blob/master/target/linux/archs38/image/Makefile):
```
define Image/Prepare
	# Build .dtb for all boards we may run on
	$(foreach dts,$(DEVICE_DTS_LIST),
		$(call Image/BuildDTB,$(DTS_DIR)/$(dts).dts,$(DTS_DIR)/$(dts).dtb)
	)
endef
```

I run clean build for fsl-ls1012a-frwy-sdboot and it works.